samples/multiplatform/build.gradle

```diff
@@ -11,13 +11,16 @@ apply plugin: 'maven-publish'
 
 kotlin {
     jvm()
-    js()
+    js {
+        nodejs {
+        }
+    }
     sourceSets {
         commonTest {
             dependencies {
                 implementation kotlin('test-common')
                 implementation kotlin('test-annotations-common')
-                implementation ('ch.tutteli.atrium:atrium-fluent-en_GB-common:0.9.0-alpha2')
+                implementation('ch.tutteli.atrium:atrium-fluent-en_GB-common:0.9.0-alpha2')
```

Comment:
```suggestion
                implementation('ch.tutteli.atrium:atrium-fluent-en_GB-common:0.9.1')
```

Reply:
same same but different for other dependencies. Maybe it is best if you extract the versino number to a variable